services/lsst-dev.rst

```diff
@@ -66,6 +66,8 @@ Enter your passphrase at the prompts:
    The key fingerprint is:
    a1:b2:c3:45:67:89:d1:e2:f3:54:76:98:00:aa:bb:01 username@hostname.lsstcorp.org
 
+If you used a program other than OpenSSH for this step, make sure your public key is formatted as a single line (most SSH clients provide it as an option). Otherwise, the next step will not work.
```

Comment:
This paragraph might work well inside a `.. note::` directive. See http://www.sphinx-doc.org/en/stable/markup/para.html?highlight=note#directive-note